# Medizinische Genomanalysen 2019
Material and information about the lecture "Medizinische Genomanalysen" 2019

## Overview
* Fork & clone the repository of each assignment
* Each student needs to work on his/her assignment (no team work)
* When done, commit and push the results to your github repository
* All assignments need to be completed by **17.05.2019**

## Python packages
* Biopython (http://biopython.org/)
* pybedtools (https://daler.github.io/pybedtools/)
* NCBI (http://biopython.org/DIST/docs/api/Bio.Entrez-module.html)
* pybam (https://github.com/JohnLonginotto/pybam)
* pysam (http://pysam.readthedocs.io/en/latest/usage.html)
* cyvcf2 (https://github.com/brentp/cyvcf2)
* pyvcf (http://pyvcf.readthedocs.io/en/latest/)
* HGVS - does only work with Python2! (https://hgvs.readthedocs.io/en/master/)
* pyfaidx



## Assignment 1
Repository: [medizinische_genomanalysen_2019_assignment_1](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_1)

## Assignment 2
Repository: [medizinische_genomanalysen_2019_assignment_2](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_2)

## Assignment 3
Repository: [medizinische_genomanalysen_2019_assignment_3](https://github.com/spabinger/medizinische_genomanalysen_2019_assignment_3)



## Student projects

### Exercise - Clinical sequencing

#### Your TODOs
1. Per group present the selected software
   * Software: 10min 
2. Perform 6 analysis steps (if possible) using FASTQ files (use only one file and chr21 to speed up the analysis)
3. Present the performed analysis


#### Perform the following steps (if supported)
1. Data Import – take FASTQ files from:
   * *Illumina:  ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data*
   * *Ion Torrent: (Only for Ion Reporter)  ftp://ftp-trace.ncbi.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/ion_exome/*
   * *Requirement: human, genome (no RNASeq, methylation, …)*
2. Quality Control & QC Report
3. Mapping
4. Variant Calling (SNV, Indel)
5. Variant Annotation
6. Interpretation & Decision Making

#### Student groups 
1. Illumina basespace  --  https://basespace.illumina.com/home/index 
   * Arnold	Vivienne
   * Benkö	Alexander
   * Bilek	Gerhard
   * Brenner	Stefan
   * Erhart	Ricarda

2. Maser  --  http://cell-innovation.nig.ac.jp/maser/index_en.html
   * Fischer	Johanna
   * Glück	Tobias
   * Gorki	Anna-Dorothea
   * Hobiger	Johanna
   * Huber	Lukas
3. Galaxy  --  https://main.g2.bx.psu.edu/ 
   * Jansen	Christian
   * Kriz	Richard
   * Lehner	Eva Valerie
   * Lenz	René
   * Luger	Martina
4. Chipster  --  http://chipster.csc.fi/ 
   * Maxymovitz	Eva
   * Meitz	Sarah
   * Mock	Michael
   * Molin	Alexander
   * Prem	Max
   * Ropek	Nathalie
5. GenePattern  --  https://cloud.genepattern.org/
   * Schneider	Lisa
   * Sehr	Eva Maria
   * Turkusic	Sabina
   * Umlauf	Ellen
   * Viehböck	Dominic
   * Zajac	Marek


*Backup: https://www.basepairtech.com/*








